pkgs/development/python-modules/kotsu/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pandas
+, typing-extensions
+, pytestCheckHook
+, pytest-mock
+, pytest-cov
+, scikit-learn
+}:
+
+buildPythonPackage rec {
+  pname = "kotsu";
+  version = "0.3.3";
+  format = "setuptools";
+
+  disable = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "datavaluepeople";
+    repo = "kotsu";
+    rev = "v${version}";
+    hash = "sha256-V5OkgiLUTRNbNt6m94+aYUZd9Nw+/60LfhrqqdFhiUw=";
+  };
+
+  propagatedBuildInputs = [ pandas typing-extensions ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-mock
+    pytest-cov
```

Comment:
```suggestion
```
We do not need to measure coverage, because we run tests to verify that the package works appropriately.

Reply:
Got it, in this case we should also remove coverage flags in `setup.cfg`. Removed `pytest-cov` and added patch to remove flags